src/model/range.js

```diff
@@ -462,6 +462,27 @@ export default class Range {
 				// <p>foo</p><p>[]bar{}</p> -> <p>foo[]bar{}</p>
 				ranges[ 0 ] = new Range( targetPosition.getShiftedBy( this.start.offset ) );
```

Comment:
It'd be nice to change this to `return [ new Range( ... ) ]`, to fit versions below.